client/src/components/ui/with-ellipsis.tsx

```diff
@@ -1,31 +1,37 @@
 import { useState } from 'react';
 
-import { Tooltip, TooltipContent, TooltipTrigger, TooltipArrow } from '@radix-ui/react-tooltip';
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipTrigger,
+  TooltipArrow,
+  TooltipProvider,
+} from '@/components/ui/tooltip';
 
 const WithEllipsis = ({ text, maxLength = 35 }: { text: string; maxLength?: number }) => {
   const [tooltipOpen, setTooltipOpen] = useState(false);
+  const handleTooltipClick = () => setTooltipOpen((prevOpen) => !prevOpen);
 
   if (text?.length < maxLength) return text;
 
   const triggerText = text && `${text.slice(0, maxLength)}...`;
   return (
-    <Tooltip open={tooltipOpen}>
-      <TooltipTrigger
-        className="cursor-default"
-        onMouseEnter={() => setTooltipOpen(true)}
-        onMouseLeave={() => setTooltipOpen(false)}
-      >
-        {triggerText}
-      </TooltipTrigger>
-      <TooltipContent
-        align="center"
-        side="bottom"
-        className="z-50 max-w-[300px] bg-white p-1 text-center text-xs text-slate-700"
-      >
-        {text}
-        <TooltipArrow className="z-50 fill-secondary" width={10} height={5} />
-      </TooltipContent>
-    </Tooltip>
+    <TooltipProvider delayDuration={0}>
```

Comment:
This tooltip doesn't show when the trigger receives focus, while the other does. Can you standarise?

Reply:
Also, the icon is not part of the trigger here while it is for the other.

Reply:
@Bluesmile82 I think you forgot my previous message. On the cards, the initiative type's icon is part of the trigger but the regions' icon is not.

Reply:
Thanks! I think I missed that one. Ready now